packages/nimble-components/src/icon-base/tests/icons.stories.ts

```diff
@@ -1,22 +1,33 @@
 import type { Meta, StoryObj } from '@storybook/html';
-import { html, repeat } from '@microsoft/fast-element';
+import { html, ref, repeat } from '@microsoft/fast-element';
 import { DesignSystem } from '@microsoft/fast-foundation';
 import * as nimbleIconComponentsMap from '../../icons/all-icons';
 import { IconSeverity } from '../types';
 import {
     createUserSelectedThemeStory,
     overrideWarning
 } from '../../utilities/tests/storybook';
-import type { Icon } from '..';
 import {
     tokenNames,
     scssInternalPropertySetterMarkdown
 } from '../../theme-provider/design-token-names';
+import { Table, tableTag } from '../../table';
+import { tableColumnIconTag } from '../../table-column/icon';
+import { mappingIconTag } from '../../mapping/icon';
+import { tableColumnTextTag } from '../../table-column/text';
+import { iconMetadata } from './icon-metadata';
 
-const nimbleIconComponents = Object.values(nimbleIconComponentsMap);
+type IconName = keyof typeof nimbleIconComponentsMap;
+const data = Object.values(nimbleIconComponentsMap).map(klass => ({
```

Comment:
Just curious `klass` is an unreserved keyword version of `class`? Or something else?

Reply:
yep 😅 something deep inside my finger memory, but I wouldn't let that through review these days so good catch, Updated to a normal name of iconClass